src/Client/Web/Bit.Client.Web.BlazorUI/Components/Rating/BitRating.razor.cs

```diff
@@ -24,13 +26,27 @@ public bool IsReadonly
             }
         }
 
+        [Parameter]
+        public int Value
+        {
+            get => ratingValue;
+            set
+            {
+                ratingValue = value;
+                ClassBuilder.Reset();
+
+                _ = ValueChanged.InvokeAsync(value);
+            }
+        }
+
         public string[] RatingColorClass { get; set; }
         public string[] RatingIcon { get; set; }
 
         private string colorClass { get; set; } = "bit-rating-dark-fluent";
         private int min { get; set; }
 
         private bool isReadOnly;
+        private int ratingValue;
```

Comment:
move these property private fields at the top of the class.
just like BitCheckBox component

Reply:
this one was not fixed eiter